server/controler/login_test.go

```diff
@@ -61,5 +64,14 @@ func (suite *LoginReqTestSuite) TestRejectWhenBodyIsNotJson() {
 	assert.Equal(suite.T(), http.StatusBadRequest, suite.rr.Code)
 }
 
+func (suite *LoginReqTestSuite) TestRejectIfUserNofFoundInDataStore() {
+	jsonBody, _ := json.Marshal(map[string]string{"username": "foo", "password": "bar"})
```

Comment:
I think it'd be better to export `loginReq` type and use it here

Reply:
you're 100% right